blocks/header/header.js

```diff
@@ -114,18 +146,52 @@ export default async function decorate(block) {
     brandLink.closest('.button-container').className = '';
   }
 
+  // populate logo
+  brandLink.remove();
+  navBrand.appendChild(createNestedDivs(['group', 'logo'], 2));
+  const image = document.createElement('img');
+  image.classList.add('fill');
+  image.setAttribute('src', '/icons/logo.svg');
+  navBrand.querySelector('.group').appendChild(image);
```

Comment:
Icons should be added as span and decorated by https://github.com/urfuwo/hlx-test/blob/main/scripts/aem.js#L419